### PR TITLE
wordpress and plugin updates

### PR DESCRIPTION
**Pull request by stephdavid**
Update WordPress to latest version 5.3.2  and plugin Google Analytics Dashboard ^5.3.10

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
         "php": ">=7.1",
         "composer/installers": "^1.4",
         "vlucas/phpdotenv": "^2.0.1",
-        "johnpbloch/wordpress": "5.2.4",
+        "johnpbloch/wordpress": "5.3.2",
```
Comment (wilson1000-MoJ) [inline composer.json]: Hi Stephanie!!! :D 
Thank you. Would you kindly replace the version number with a * and run composer update again?

File: composer.json [hunk 2]
```diff
@@ -68,4 +68,4 @@
             "vendor/bin/phpcs"
         ]
     }
-}
+}
```
Comment (wilson1000-MoJ) [inline composer.json]: Would you kindly add an new line at the end of the file please? The reason is to be safe; some older software applications won't work correctly if the last line is not terminated with a new line. Thank you.

File: composer.json [hunk 3]
```diff
@@ -34,7 +34,7 @@
         "wpackagist-plugin/fast-user-switching": "*",
         "wpackagist-plugin/wp-browser-update": "*",
         "relevanssi/relevanssi-premium": "*",
-        "wpackagist-plugin/google-analytics-dashboard-for-wp": "^5.3",
+        "wpackagist-plugin/google-analytics-dashboard-for-wp": "^5.3.10",
```
Comment (wilson1000-MoJ) [inline composer.json]: The ^ symbol at the start will auto-load the .10 version without having to implicitly define it. Have you found an incompatibility with a newer version?

**Review by wilson1000-MoJ — CHANGES_REQUESTED** [review]
Changes required. Thank you :)

**Comment by wilson1000-MoJ** [comment]
Also, please commit the compose.lock file too (you will probably do this anyway) as we need this to lock out versions during deploy with dependency updates via *

**Comment by stephdavid** [comment]
Certainly made a mess of this - I tried a couple of extensions to Visual Studio Code to add the final line / or not remove it (which it seems to do by default). I resorted to using a text editor and basic terminal to push the changes. In the meantime, will keep trying to sort out VSC.

**Review by wilson1000-MoJ — APPROVED** [review]
Looks good, thank you 🙏

**Pull request merged**